symfony/src/Service/RdrApiService.php

```diff
@@ -14,19 +16,27 @@ class RdrApiService
     protected $endpoint = 'https://pmi-drc-api-test.appspot.com/';
     protected $config = [];
     protected $cache;
+    protected $logger;
 
-    public function __construct(EnvironmentService $environment, KernelInterface $appKernel, GoogleClient $googleClient, ParameterBagInterface $params)
+    public function __construct(EnvironmentService $environment, KernelInterface $appKernel, GoogleClient $googleClient, ParameterBagInterface $params, LoggerInterface $logger)
     {
         $this->googleClient = $googleClient;
         $basePath = $appKernel->getProjectDir();
         // Note that when installed in ./symfony, the development credentials are a level down
         if ($environment->isLocal() && file_exists($basePath . '/../dev_config/rdr_key.json')) {
             $this->config['key_file'] = $basePath . '/../dev_config/rdr_key.json';
         }
+        if ($params->has('rdr_auth_json') && !$params->has('rdr_auth_json_disabled')) {
```

Comment:
The "kill switch" I had in mind wasn't to disable the use of the auth json configuration, but instead to disable the use of caching. The caching is what caused problems last time and it would be nice to be able to disable it without a code change.

Reply:
Resolved in ee1a7d29